### PR TITLE
Add 'Marketplace' and 'Explore' to the defaults

### PR DESCRIPTION
**Pull request by darkred**
This PR makes the following changes:

- adds in the default GitHub and Gists links the following: `"Marketplace", "Explore"`.
My reasoning:
by default the Navigation bar is: 

I believe the 'Marketplace' and 'Explore' are important links (especially the latter), the former to find useful tools (free for public repos) to build on and improve your workflow, and the latter to find collections/topics/repos/resources that match your interests or are popular/trending.
If they aren't provided by the default, the user will probably will never discover them by himself.

- updates the ''Integrations" link:  https://github.com/integrations --> https://github.com/marketplace 
- removes the 'Trending' entry, as its content is now contained inside the 'Explore' page, I've noticed.

@Mottie, what do you think with my suggested changes?

### CODE REVIEW

**Comment by Mottie** [comment]
Hi @darkred!

I like it! But, there are a few issues that need fixing before we merge.

File: github-custom-navigation.user.js [hunk 1]
```diff
@@ -82,8 +82,8 @@
 					content: "<svg class='octicon' xmlns='http://www.w3.org/2000/svg' aria-hidden='true' height='16' viewBox='0 0 16 16' width='16'><path d='M14.7 5.34c.13-.32.55-1.59-.13-3.31 0 0-1.05-.33-3.44 1.3-1-.28-2.07-.32-3.13-.32s-2.13.04-3.13.32c-2.39-1.64-3.44-1.3-3.44-1.3-.68 1.72-.26 2.99-.13 3.31C.49 6.21 0 7.33 0 8.69 0 13.84 3.33 15 7.98 15S16 13.84 16 8.69c0-1.36-.49-2.48-1.3-3.35zM8 14.02c-3.3 0-5.98-.15-5.98-3.35 0-.76.38-1.48 1.02-2.07 1.07-.98 2.9-.46 4.96-.46 2.07 0 3.88-.52 4.96.46.65.59 1.02 1.3 1.02 2.07 0 3.19-2.68 3.35-5.98 3.35zM5.49 9.01c-.66 0-1.2.8-1.2 1.78s.54 1.79 1.2 1.79c.66 0 1.2-.8 1.2-1.79s-.54-1.78-1.2-1.78zm5.02 0c-.66 0-1.2.79-1.2 1.78s.54 1.79 1.2 1.79c.66 0 1.2-.8 1.2-1.79s-.53-1.78-1.2-1.78z'/></svg>"
 				},
 				"integrations": {
```
Comment (Mottie) [inline github-custom-navigation.user.js]: Please change this to "marketplace". I'm seeing "undefined" when I do a factory reset on the userscript.

File: github-custom-navigation.user.js [hunk 2]
```diff
@@ -82,8 +82,8 @@
...
-					url: "https://github.com/integrations",
-					tooltip: "Integrations",
+					url: "https://github.com/marketplace",
+					tooltip: "Marketplace",
 					hotkey: "",
 					content: "<svg class='octicon' xmlns='http://www.w3.org/2000/svg' aria-hidden='true' height='16' viewBox='0 0 14 16' width='14'><path d='M3 6c-.55 0-1 .45-1 1v2c0 .55.45 1 1 1h8c.55 0 1-.45 1-1V7c0-.55-.45-1-1-1H3zm8 1.75L9.75 9h-1.5L7 7.75 5.75 9h-1.5L3 7.75V7h.75L5 8.25 6.25 7h1.5L9 8.25 10.25 7H11v.75zM5 11h4v1H5v-1zm2-9C3.14 2 0 4.91 0 8.5V13c0 .55.45 1 1 1h12c.55 0 1-.45 1-1V8.5C14 4.91 10.86 2 7 2zm6 11H1V8.5c0-3.09 2.64-5.59 6-5.59s6 2.5 6 5.59V13z'></path></svg>"
```
Comment (Mottie) [inline github-custom-navigation.user.js]: Please replace the marketplace icon with this one I put together, if you think it looks good:

```html
<svg class='octicon' xmlns='http://www.w3.org/2000/svg' aria-hidden='true' width='16' height='16'><path d='M4 5l2 2m2.5-2l2 2m3 0l1-1L12 3H2.5M8 15.5H.5V.5H12v1m.6 8.1a3 3 0 0 1 3 3 3 3 0 0 1-3 3 3 3 0 0 1-3-3 3 3 0 0 1 3-3zm-2.8 1.8l-.7-.8 1.3-1.2.6.6m1.4 1v1.7m1.3 0h-1.3' fill='none' stroke='currentColor'/></svg>
```

**Comment by Mottie** [comment]
And maybe replace the explore icon with the trending icon? The megaphone doesn't really fit as an explore button... I'm not sure why I picked it for "explore" in the first place 😉

**Comment by darkred** [comment]
Added your suggestions (and sorry about the 1st one, it missed my attention).
And, yes, I like the marketplace icon 👍

**Comment by Mottie** [comment]
Thanks!

**Comment by Mottie** [comment]
I'm just putting the finishing touches on another userscript, then I'll double check if everything is updated.

**Pull request merged**